gym_duckietown_agent/envs/simplesimagent_env.py

```diff
@@ -22,7 +20,7 @@ class SimpleSimAgentEnv(gym.Env):
         'video.frames_per_second': 30  # TODO: do we need this on the client?
     }
 
-    def __init__(self, silent=False):
+    def __init__(self, silent, camera_width, camera_height):
```

Comment:
@AndreaCensi why take away False from silent=False - will it fail if called without a 2nd parameter now ?

Reply:
I should have changed the calls. But don't have strong feelings about it.

(For me default arguments should carry the assurance that the defaults never change. This in practice is not respected so better make everything explicit.)